From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 9 Aug 2025 19:37:07 +0200
Subject: search: Don't install library
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

We don't want to ship another shared library

Signed-off-by: Guido Günther <agx@sigxcpu.org>
Part-of: <https://gitlab.gnome.org/World/Phosh/phosh/-/merge_requests/1755>
(cherry picked from commit be23d40d48fc86be41e39e82276e56a931cc22b4)
---
 src/search/meson.build | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/search/meson.build b/src/search/meson.build
index 585bea1..3a255f5 100644
--- a/src/search/meson.build
+++ b/src/search/meson.build
@@ -19,7 +19,7 @@ phosh_search_lib = static_library(
   include_directories: [root_inc, dbus_inc],
   dependencies: phosh_search_deps,
   install_dir: pkglibdir,
-  install: true,
+  install: false,
 )
 phosh_search_inc = include_directories('.')

